supabase/migrations/20260717234851_eve_governance_kernel.sql

```
@@ -0,0 +1,84 @@
+    initiated_by_profile_id UUID
+        REFERENCES public.profiles(id)
+        ON DELETE SET NULL,
+    accountable_trigger TEXT,
+    summary JSONB NOT NULL DEFAULT '{}'::jsonb
+        CHECK (jsonb_typeof(summary) = 'object'),
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
+);
+
+CREATE INDEX eve_run_summaries_status_updated_idx
+    ON public.eve_run_summaries (status, updated_at DESC);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)